PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+#Azure SDK for Java Pull Request Template Guide
+
+Thank you for your interest in contributing to Azure SDK for Java.
+
+## Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change with fixes an issue)
+- [ ] New feature (non-breaking change with adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
+- [ ] This change requires a documentation update
+
+## How has this been tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
+
+Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
```

Comment:
where did this template come from?  We don't care about firmware version 😄

Reply:
It is the standard template found online